src/components/GarminUploadInstructions/GarminUploadInstructions.tsx

```diff
@@ -58,43 +59,43 @@ export function GarminUploadInstructions({
     setCookie(GARMIN_UPLOAD_INSTRUCTIONS_OPEN_COOKIE, newValue);
   };
 
+  const isMediumDisplayOrLarger = useDisplayIsSizeOrLarger('md');
+
   return (
-    <Stack spacing={1}>
```

Comment:
This outer stack only had one element inside it 👻

Reply:
Humiliating...